packages/chat-component-bindings/src/errorBarSelector.ts

```diff
@@ -20,8 +21,64 @@ export const errorBarSelector = createSelector([getLatestErrors], (latestErrors)
   //
   // We chose to stable sort by error type: We intend to show only a small number of errors on the UI and we do not
   // have timestamps for errors.
-  if (latestErrors['ChatThreadClient.sendMessage'] !== undefined) {
-    return { activeErrors: ['sendMessageGeneric'] };
+  const activeErrors: ErrorType[] = [];
+  let specificSendMessageErrorSeen = false;
+  if (hasUnableToReachBackendError(latestErrors)) {
+    activeErrors.push('unableToReachChatService');
```

Comment:
Possible to have consistent naming?
`hasUnableToReachChatServiceError` and `unableToReachChatService`

Reply:
This function checks if `latestErrors` contains the error `unableToReachChatService`.

Would the name `containsUnableToReachChatService` be better?

Reply:
I think `has` prefix is ok. I was just concerned with different choice of words.
Method is `hasUnableToReachBackendError`
and we are pushing a string `unableToReactChatService`

Suggestion is to convert the method to `hasUnableToReactChatServiceError`